src/helpers/sceneless-project.ts

```diff
@@ -2215,6 +2287,22 @@ export const createCompositor = async (
       },
       root.id,
     ),
+    project.insert(
+      {
+        id: 'audio',
+        name: 'AudioContainer',
+        layout: 'Free',
+        size: {
+          x: 0,
+          y: 0,
+        },
+        position: {
+          x: 0,
+          y: 0,
+        },
+      },
+      root.id,
+    ),
   ])
```

Comment:
In my opinion (and based on my experience while working on this), we don't want it to be part of the content layer, because that will affect the layout of the content node. That's really the whole purpose of the `AudioContainer` as a separate place for these nodes. I'll change it to make sure it's on the bottom (good call on that), and also make sure it's `opacity: 0` just to be safe. if necessary. If it ends up being a problem we can fix it then, but 1 day before launch I'd rather leave it as is.